src/Symfony/Component/Process/Pipes/UnixPipes.php

```diff
@@ -74,7 +74,7 @@ public function getDescriptors(): array
             return [
                 ['pty'],
                 ['pty'],
-                ['pty'],
+                ['pipe', 'w'], // stderr need to be in a pipe to correctly split error and output, since PHP will use the same stream for both
```

Comment:
```suggestion
                ['pipe', 'w'], // stderr needs to be in a pipe to correctly split error and output, since PHP will use the same stream for both
```